tests/bson/bson-toPHP-004.phpt

```diff
@@ -553,7 +553,7 @@ object(stdClass)#%d (1) {
 object(stdClass)#%d (1) {
   ["x"]=>
   object(stdClass)#%d (1) {
-    [%r(0|"0")%r]=>
+    ["0"]=>
```

Comment:
I think you got everything in this file (they were all `%r(0|"0")%r` AFAICT), but searching the project for `")%r]` will catch everything.